src/python/example/alpha_complex_from_generated_points_on_sphere_example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+from gudhi.datasets.generators.points import sphere
+from gudhi import AlphaComplex
+
+
+""" This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+    Author(s):       Hind Montassif
+
+    Copyright (C) 2021 Inria
+
+    Modification(s):
+      - YYYY/MM Author: Description of the modification
+"""
+
+__author__ = "Hind Montassif"
+__copyright__ = "Copyright (C) 2021 Inria"
+__license__ = "MIT"
+
+print("#####################################################################")
+print("AlphaComplex creation from generated points on sphere")
+
+
+# Generate a circle: 50 points; dim 2; radius 1
+points = sphere.generate_random_points(50, 2, 1)
```

Comment:
Yes indeed, that was intended to resume discussion on the naming matter. Since the `points` module is preferred to be before `sphere` and that `random` is more specific to `points` than `sphere`, I would say let's go with a parameter for now.